### PR TITLE
Fix encoding on Photo Details page

### LINKED ISSUE #192: Fix encoding on Photo Details page
Decode the data on the front-end, so that if any data comes encoded in the API results, users won't see strange characters on the page.

Link to original issue https://github.com/creativecommons/cccatalog/issues/107

### PR DESCRIPTION
**Pull request by VatsalSin**
-> Decodes the data encoded in the API results on the front-end.
-> Renders "Léontine Walter" instead of "L\xe9ontine Walter". 
-> Problem Detected:- The characters in data returned by API are double escaped.

Fixes #192, #181, #154 and #107 

screenshot 54

### CODE REVIEW

**Comment by VatsalSin** [comment]
Required changes done.

File: test/unit/specs/utils/decodeData.spec.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    const data = undefined;
+    const empty="";
+
+    expect(decodeData(data)).toEqual(""); 
+  });
+
+  it('decodes unicode strings', () => {
+    const data = 's\\xe9'
+
+    expect(decodeData(data)).toBe('s\xe9');
```
Comment (brenoferreira) [inline test/unit/specs/utils/decodeData.spec.js]: Did you also test this with strings with unicode characters such as `'string \u1234'`?

Comment (VatsalSin) [reply test/unit/specs/utils/decodeData.spec.js]: Added decoding for unicode characters and tests for the same.
